### PR TITLE
Community hub for microsite

### PR DESCRIPTION
**Pull request by suuus**
## Hey, I just made a Pull Request!

To make it easier to find all things community, I've created this PR which will add a community hub, video archive, and a live stream page to backstage.io. 

- Adding /community: A community hub to list all things community on the website. This will not phase out backstage/community repo, since this will be used for interactions around the community sessions.
- Adding /on-demand: A page where upcoming and past community sessions (and other community recordings) can be shared and found.
- Adding /nominate: A page with the embedded contributor Spotlight nomination form
- Adding /live: A page where we can share our community sessions to, using a Youtube embed which we'll need to update after scheduling a live stream.

/on-demand can be configured with yaml files in /data/on-demand (a blatant copy of the plugins page)

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/pages/en/community.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+const React = require('react');
+const Components = require(`${process.cwd()}/core/Components.js`);
+const Block = Components.Block;
+
+const Background = props => {
+  const { config: siteConfig } = props;
+  const { baseUrl } = siteConfig;
+  return (
+    <main className="MainContent">
+      {new Date('2022-03-17') > new Date() ? <Component /> : null}
```
Comment (freben) [inline microsite/pages/en/community.js:17]: Should delete this now

File: microsite/pages/en/community.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+      <Block small className="stripe-bottom bg-black-grey">
+        <Block.Container style={{ justifyContent: 'flex-start' }}>
+          <Block.Title>Backstage Community</Block.Title>
+          <Block.TextBox>
+            <Block.Paragraph>
+              What's the use of having fun if you can't share it? Exactly. Join
+              the vibrant community around the Backstage project. Be it on
+              GitHub, social media, Discord.. You'll find a welcoming
```
Comment (freben) [inline microsite/pages/en/community.js:26]:
```suggestion
              GitHub, social media, Discord... You'll find a welcoming
```

File: microsite/pages/en/community.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      <Block small className="stripe bg-black">
+        <Block.Container style={{ justifyContent: 'flex-start' }}>
+          <Block.Title>Community Sessions</Block.Title>
+          <Block.TextBox>
+            <Block.Paragraph>
+              <b>Adopters Community Sessions</b>
+              <br />
+              Backstage Community Sessions is the monthly meetup where we all
+              come together to listen to the latest maintainer updates, learn
+              from eachother about adopting, share exciting new demo's or
```
Comment (freben) [inline microsite/pages/en/community.js:67]:
```suggestion
              from eachother about adopting, share exciting new demos or
```

File: microsite/pages/en/community.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+              <br />
+            </Block.Paragraph>
+            <Block.LinkButton href="nominate ">Nominate now</Block.LinkButton>
+          </Block.TextBox>
+          <Block.TextBox>
+            <Block.Title>Open Mic Meetup</Block.Title>
+            <Block.Paragraph>
+              A monthly casual get together of Backstage users sharing their
+              experiences and helping each other. Hosted by Roadie.io and
+              Frontside Software.
```
Comment (freben) [inline microsite/pages/en/community.js:130]: might be nice to link to their respective home pages here?

File: microsite/pages/en/on-demand.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+                  </a>
+                  <a className="VideoButtonFilled" href={rsvpUrl}>
+                    Remind me
+                  </a>
+                </div>
+              </div>
+            ),
+          )}
+      </Container>
+      <BulletLine style={{ width: '100% ' }} />
```
Comment (freben) [inline microsite/pages/en/on-demand.js:81]:
```suggestion
      <BulletLine style={{ width: '100%' }} />
```

**Review by freben — COMMENTED** [review]
Alright, just some small nits

File: microsite/pages/en/community.js (hunk 4 above)
Comment (suuus) [reply microsite/pages/en/community.js:130]: 🤦‍♀️ yes!

File: docs/overview/support.md [hunk 6]
```diff
@@ -18,3 +18,47 @@ description: Support and Community Details and Links
...
+### Adding a recording to the meetup page
+
+To add a new recording to the [meetup page](https://backstage.io/on-demand)
+create a file in
+[`microsite/data/on-demand`](https://github.com/backstage/backstage/tree/master/microsite/data/on-demand)
+with your recording's information. Example:
+
+```yaml
+---
+---
```
Comment (freben) [inline docs/overview/support.md:35]: seems you got double separator lines here and below 🤔

**Review by freben — APPROVED** [review]
Alright, one newly introduced thing, but approving - ready to merge when those double lines are gone!

**Pull request merged**